src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1041,6 +1041,7 @@ void cbmc_parse_optionst::help()
     "\n"
     "Backend options:\n"
     " --object-bits n              number of bits used for object addresses\n"
+    // NOLINTNEXTLINE(whitespace/line_length)
```

Comment:
The commit message says "will be squashed" so presumably that should happen ;-)

Reply:
Squashed.